django_elasticsearch_dsl/documents.py

```diff
@@ -191,9 +191,9 @@ def update(self, thing, refresh=None, action='index', parallel=False, **kwargs):
         """
         Update each document in ES for a model, iterable of models or queryset
         """
-        if refresh is True or (
-            refresh is None and self.django.auto_refresh
-        ):
+        if refresh is not None:
+            kwargs['refresh'] = refresh
+        elif self.django.auto_refresh:
```

Comment:
@chosak I think people should have the ability to set the `wait_for` in the document as well. What about setting like following?
```suggestion
        elif self.django.auto_refresh:
            kwargs['refresh'] = self.django.auto_refresh
 ```
```

Reply:
Makes sense. I've added a test for this and updated the documentation as well. The next step would be to allow `settings.ELASTICSEARCH_DSL_AUTO_REFRESH` to support `wait_for` as well...

Reply:
Sounds interesting! I need to think the naming as well. Because `auto_refresh` does not make sense anymore. But we can think about it in future. 💯